addons/colored-follower-projects/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Colored projects made by people you follow",
+  "description": "Colors any project on the home page or 'Explore' that was made by someone you follow.",
```

Comment:
"highlights" instead of "colors"

Specifiy it works on the search too

The title is too long IMO

@mxmou is this consistent with #2799?